template/.electron-vue/dev-runner.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+process.env.NODE_ENV = 'production'
```

Comment:
I don't think it's right way to fix this bug.

Reply:
Thank you for reminding me. I have fixed it. Is this the desired result now?

Reply:
So stupid mistake!

Reply:
1. `process.env.NODE_ENV` is undefined in `.eslintrc.js` in development mode, because we are not passing string while linter runs, but for production it reads `production` as string, it works how it should and `.eslintrc.js` doesn't need changes.
2. Highlighted third line in your code, I think is wrong way to pass data for NODE_ENV, you can do it same way as we did for production: 
https://github.com/SimulatedGREG/electron-vue/blob/8fae4763e9d225d3691b627e83b9e09b56f6c935/template/package.json#L34
right example: 
`cross-env NODE_ENV=development webpack --progress --colors --config .electron-vue/webpack.renderer.config.js`

Conclusion:
`debugger` for development environment already works fine and currently we have only two mode for NODE_ENV: `development` & `production`, in most cases `NODE_ENV` is `undefined` because we start coding from development and then we separate environment for production.
I hope you understand idea